services/ingestion/etl/types/Project.js

```diff
@@ -41,11 +41,11 @@ type GeoJSON =
   | MultiPolygon2D;
 
 type Budget = {
-  eu_contrib: number,
-  total_cost: number,
-  private_fund: number,
-  public_fund: number,
-  other_contrib: number,
+  eu_contrib: { value: number, currency: string, raw: string },
```

Comment:
Would be nice to make `{ value: number, currency: string, raw: string }` a type ;)

Reply:
Indeed, something like

```
type Currency = {
  currency: string,
  raw: string,
  value: number
};
```

Value seems almost a reserved keyword, but it's not, let's hope it won't create any conflicts down the road.